app/jobs/clean_stalled_uphold_codes_and_access_parameters_job.rb

```diff
@@ -0,0 +1,30 @@
+class CleanStalledUpholdCodesAndAccessParametersJob < ApplicationJob
```

Comment:
minor: this name is a mouthful - perhaps `CleanStalledUpholdDataJob` or even `CleanStaleUpholdDataJob`?

Reply:
Haha yes that's better, I will make the change.